app/controllers/image_uploads_controller.rb

```diff
@@ -42,19 +45,12 @@ def cloudinary_link(uploaders)
 
   private
 
-  def image_upload(images)
-    if images.is_a? Array
-      images.map do |image|
-        uploader = ArticleImageUploader.new
+  def upload_images(images, rate_limiter)
+    Array.wrap(images).map do |image|
```

Comment:
`Array.wrap` wraps an object in an array, and does nothing if the object is already an array, this way we can get rid of the if